### PR TITLE
Keikodocs2

### PR DESCRIPTION
**Pull request by keikhara**
fix doc 
@samisms

### CODE REVIEW

File: docs/OMS-Agent-for-Linux.md [hunk 1]
```diff
@@ -132,19 +132,24 @@ Communication between the agent and OMS services can use an HTTP or HTTPS proxy
 
 **Proxy Configuration**
 The proxy configuration value has the following syntax:
-[protocol://][user:password@]proxyhost[:port]
+[protocol://][user:password@]proxyhoddst[:port]
```
Comment (lagalbra) [inline docs/OMS-Agent-for-Linux.md]: This seems like a typo...

File: docs/OMS-Agent-for-Linux.md [hunk 2]
```diff
@@ -132,19 +132,24 @@ Communication between the agent and OMS services can use an HTTP or HTTPS proxy
...
 
 Property|Description
 -|-
 Protocol|http or https
-user|Optional username for proxy authentication
-password|Optional password for proxy authentication
+user|User for proxy authentication; can be placeholder text
```
Comment (lagalbra) [inline docs/OMS-Agent-for-Linux.md]: Rebase against master; these changes were already made.

Comment (lagalbra) [inline docs/OMS-Agent-for-Linux.md]: Correction; these changes were partially made. But you must rebase this branch against master to get the correct diff.

File: docs/OMS-Agent-for-Linux.md (hunk 1 above)
Comment (lagalbra) [inline docs/OMS-Agent-for-Linux.md]: Please fix this typo, and then merge soon.

File: docs/OMS-Agent-for-Linux.md [hunk 3]
```diff
@@ -137,14 +137,19 @@ The proxy configuration value has the following syntax:
...
+password|Password for proxy authentication; can be placeholder text
 proxyhost|Address or FQDN of the proxy server
-port|Optional port number for the proxy server
+port|Port number for the proxy server
 
 For example:
 http://user01:password@proxy01.contoso.com:8080
 
+*Note: Although you do not have any user/password set for the proxy, you will still need to add a psuedo user/password. This can be any username or password. (In the future release, this pseudo settings will not be mandatory.)*
+
```
Comment (lagalbra) [inline docs/OMS-Agent-for-Linux.md]: nit: it's spelled pseudo; fix the spellings on other lines to this spelling

**Comment by lagalbra** [comment]
@Microsoft/omsagent-devs

**Comment by keikhara** [comment]
@Microsoft/omsagent-devs  fix docs

File: docs/OMS-Agent-for-Linux.md [hunk 4]
```diff
@@ -137,16 +137,19 @@ The proxy configuration value has the following syntax:
...
+user|User for proxy authentication; can be placeholder text
+password|Password for proxy authentication; can be placeholder text
 proxyhost|Address or FQDN of the proxy server
-port|Optional port number for the proxy server
+port|Port number for the proxy server
 
 For example:
 http://user01:password@proxy01.contoso.com:8080
 
-*Note: Although you do not have any user/password set for the proxy, you will still need to add a psuedo user/password. This can be any username or password.
```
Comment (lagalbra) [inline docs/OMS-Agent-for-Linux.md]: Lines 148 and 149 are still misspelled as psuedo

**Comment by keikhara** [comment]
@Microsoft/omsagent-devs

**Pull request closed without merging**